akka-javasdk/src/main/java/akka/javasdk/JsonSupport.java

```diff
@@ -96,7 +98,10 @@ private JsonSupport() {
    * for the JSON type instead.
    *
    * @see {{encodeJson(T, String}}
+   *
+   * @deprecated Protobuf Any with JSON is not supported
```

Comment:
Does it still make sense for message broker messages?
I'd say that then it's better to use a more explicit protobuf message that carry the raw bytes and type hint.

When we add "real" protobuf serialization support I think we should revisit this. Would be better to have the serialization support as something that is injected rather than static methods.

Reply:
_if_ we should add protobuf as internal serialization format at all, we can trigger it by the fact that there is a Java protobuf message base types, that way we wont need explicit user serialization like this.


BTW I think we can drop these completely, as there is no way to use them that makes sense in the new SDK, they exist because of basing off of old SDK but they could never really be used here.

The only use case where this class can be used for anything useful right now, afaics, is to do streaming JSON encoding for results in a query in a HTTP endpoint.

Reply:
I think protobuf can make sense when we add gRCP endpoints. Since that will happen later I think we can leave these as deprecated now and introduce proper serialization support at that point (and then remove these here).